workflow/commons/src/main/java/com/datastax/oss/dsbulk/workflow/commons/settings/LogSettings.java

```diff
@@ -191,6 +193,7 @@ public void init() throws IOException {
         setNormal();
       }
       this.verbosity = Verbosity.values()[verbosity];
+      printSources = config.getBoolean(PRINT_SOURCES);
```

Comment:
Why this discrepancy between settings `PRINT_SOURCES` and parameter: `retainRecordSources`? Can we unify that and name the same way?

Reply:
Yes I agree it's weird. The fact is that internally, the most visible effect is on memory, which is why I chose `retainRecordSources`; but from the user perspective, the only visible effect is the fact that the source is printed or not – which si why I chose `printSources`.

I think I will simplify the config option to just `sources`. It's not a setting that will be changed often anyways.